smithy-aws-traits/src/main/java/software/amazon/smithy/aws/traits/HttpChecksumTraitValidator.java

```diff
@@ -185,11 +185,13 @@ private List<ValidationEvent> validateHeaderConflicts(OperationShape operation,
                     .map(HttpPrefixHeadersTrait::getValue)
                     .ifPresent(headerPrefix -> {
                         if (HttpChecksumTrait.CHECKSUM_PREFIX.startsWith(headerPrefix)) {
+                            final String memberName = member.getId().getName();
```

Comment:
`final` can be removed here, except for some very rare cases we don't use `final` for method scoped variables.
```suggestion
                            String memberName = member.getId().getName();
```